src/nbsap/assets.py

```diff
@@ -7,27 +7,50 @@
     'js/uri.js',
     'bootstrap/js/bootstrap.min.js',
     'js/main.js',
+    'js/list.js',
     'js/lib/jquery.columnlist.js',
+    'bootstrap/js/sidebar-plugin.js',
+    'bootstrap/js/sidebar.js',
+    'bootstrap/js/scroll.js',
 )
 
 JS_ADMIN_ASSETS = (
+    'js/jquery-1.11.1.min.js',
+    'js/jquery.utils.js',
+    'bootstrap/js/bootstrap.min.js',
     'js/lib/datatables/jquery.dataTables.min.js',
-    'js/lib/datatables/DT_bootstrap.js',
     'js/jquery.browser.min.js',
+    'js/uri.js',
+    'js/main.js',
     'js/chosen/chosen.jquery.min.js',
+    'js/app.min.js'
 )
 
 CSS_ASSETS = (
-    'bootstrap/css/bootstrap.min.css',
+    # 'bootstrap/css/bootstrap.min.css',
+    'bootstrap/css/normalize.css',
+    'bootstrap/css/skeleton.css',
     'js/chosen/chosen.css',
     'css/style.css',
+    'https://maxcdn.bootstrapcdn.com/font-awesome/4.6.3/css/font-awesome.min.css'
```

Comment:
I'm not sure why, but when I load it locally , the app can't find the fonts. I'll leave it like this for now and set a TODO to try making it work later